### PR TITLE
bump up version from 1.5.0 to 1.6.0 on master

### PR DESCRIPTION
**Pull request by roywei**
## Description ##
bump up version from 1.5.0 to 1.6.0 on master
bump up scala `README.md` 1.3.1 to latest stable 1.4.0
### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### CODE REVIEW

File: contrib/clojure-package/README.md [hunk 1]
```diff
@@ -137,7 +137,7 @@ With this option, you will install a Git revision of the Clojure package source
 
 - Install additional dependencies as described in [the corresponding section for Option 1](#installing-additional-dependencies),
 
-- Recursively clone the MXNet repository and checkout the desired version, (example 1.3.1). You should use the latest [version](https://search.maven.org/search?q=clojure-mxnet)), and a clone into the `~/mxnet` directory:
+- Recursively clone the MXNet repository and checkout the desired version, (example 1.4.0). You should use the latest [version](https://search.maven.org/search?q=clojure-mxnet)), and a clone into the `~/mxnet` directory:
```
Comment (szha) [inline contrib/clojure-package/README.md]: should be 1.4.1 now.

File: julia/NEWS.md [hunk 2]
```diff
@@ -15,7 +15,7 @@
 <!--- specific language governing permissions and limitations -->
 <!--- under the License. -->
 
-# v1.5.0 (#TBD)
+# v1.6.0 (#TBD)
```
Comment (iblis17) [inline julia/NEWS.md]:
```suggestion
# v1.6.0


# v1.5.0
```

File: contrib/clojure-package/README.md (hunk 1 above)
Comment (roywei) [reply contrib/clojure-package/README.md]: fixed

File: julia/NEWS.md (hunk 2 above)
Comment (roywei) [reply julia/NEWS.md]: fixed

**Comment by abhinavs95** [comment]
@mxnet-label-bot add [pr-awaiting-review]

**Comment by piyushghai** [comment]
@roywei Can you resolve the merge conflicts in this PR ?

**Comment by roywei** [comment]
@mxnet-label-bot add [pr-work-in-progress]

**Pull request merged**